games-fps/redeclipse/redeclipse-1.6.0-r1.ebuild

```diff
@@ -70,10 +70,14 @@ src_install() {
 	if ! use dedicated; then
 		dobin src/redeclipse_linux
 		newicon "src/install/nix/${PN}_x128.png" "${PN}.png"
-		make_desktop_entry "src/install/nix/${PN}.desktop"
+		domenu "src/install/nix/${PN}.desktop"
 		doman doc/man/redeclipse.6
 	fi
 
+	dobin "${FILESDIR}/redeclipse"
+	cd /usr/bin
```

Comment:
`|| die`

The next command would most likely succeed in the wrong directory if the cd fails.